### PR TITLE
Company dashboard replaces navbar

### PR DESCRIPTION
**Pull request by sloschert**
I found the navbar convoluted, that's why I started to rearrange it. In the end I came up with a company dashboard that serves as the main entry point for companies and as the main menu. The navbar is very reduced now.

I think it is more organised now but that's open for discussion. 

I thought about having a navbar in addition to the dashboard but I found it unnecessary and a functional duplication.  

Plan ID: e8e5c56f-9e2d-497d-a896-a26bc41e1c16

image

### CODE REVIEW

**Comment by seppeljordan** [comment]
@sloschert Hi, I like the change that you propose. As it is right now there are some user interface elements that you touched that are not translated yet. Would it be possible to make those elements support translation?

**Comment by sloschert** [comment]
> @sloschert Hi, I like the change that you propose. As it is right now there are some user interface elements that you touched that are not translated yet. Would it be possible to make those elements support translation?

I could also help with that. But I would like to avoid merging changes which introduce new untranslatable strings. 

Hey, I forgot to make strings in dashboard.html translatable, I did it now. Regardings the rest of the (mainly html) files, I'd argue it would be beyond this PR. But I can do it in the next PR, in line with issue #312, would that be ok? I took care though not to insert any new untranslatable strings into a file that already has been made translatable.

File: arbeitszeit_flask/templates/company/transfer_to_worker.html [hunk 1]
```diff
@@ -1,5 +1,9 @@
 {% extends "base_company.html" %}
 
+{% block navbar_start %}
+<div class="navbar-item">Überweisung an Mitarbeiter</div>
```
Comment (seppeljordan) [inline arbeitszeit_flask/templates/company/transfer_to_worker.html:4]: translation

File: arbeitszeit_flask/templates/company/request_cooperation.html [hunk 2]
```diff
@@ -1,5 +1,9 @@
 {% extends "base_company.html" %}
 
+{% block navbar_start %}
+<div class="navbar-item">Kooperation anfragen</div>
```
Comment (seppeljordan) [inline arbeitszeit_flask/templates/company/request_cooperation.html:4]: translation

File: arbeitszeit_flask/templates/company/my_drafts.html [hunk 3]
```diff
@@ -1,4 +1,9 @@
 {% extends "base_company.html" %}
+
+{% block navbar_start %}
+<div class="navbar-item">Plan-Entwürfe</div>
```
Comment (seppeljordan) [inline arbeitszeit_flask/templates/company/my_drafts.html:4]: translation

File: arbeitszeit_flask/templates/company/my_cooperations.html [hunk 4]
```diff
@@ -1,5 +1,9 @@
 {% extends "base_company.html" %}
 
+{% block navbar_start %}
+<div class="navbar-item">Meine Kooperationen</div>
```
Comment (seppeljordan) [inline arbeitszeit_flask/templates/company/my_cooperations.html:4]: translation

File: arbeitszeit_flask/templates/company/list_all_cooperations.html [hunk 5]
```diff
@@ -1,5 +1,9 @@
 {% extends "base_company.html" %}
 
+{% block navbar_start %}
+<div class="navbar-item">Alle Kooperationen</div>
```
Comment (seppeljordan) [inline arbeitszeit_flask/templates/company/list_all_cooperations.html:4]: translation

File: arbeitszeit_flask/templates/company/invite_worker_to_company.html [hunk 6]
```diff
@@ -1,4 +1,9 @@
 {% extends "base_company.html" %}
+
+{% block navbar_start %}
+<div class="navbar-item">Arbeiter*in einladen</div>
```
Comment (seppeljordan) [inline arbeitszeit_flask/templates/company/invite_worker_to_company.html:4]: translation

File: arbeitszeit_flask/templates/company/create_draft.html [hunk 7]
```diff
@@ -1,4 +1,9 @@
 {% extends "base_company.html" %}
+
+{% block navbar_start %}
+<div class="navbar-item">Plan erstellen</div>
```
Comment (seppeljordan) [inline arbeitszeit_flask/templates/company/create_draft.html:4]: translation

File: arbeitszeit_flask/templates/company/account_r.html [hunk 8]
```diff
@@ -1,9 +1,15 @@
...
+<a class="navbar-item" href="{{ url_for('main_company.my_accounts') }}">{{ gettext("Accounts") }}</a>
+<div class="navbar-item">{{ gettext("Account r") }}</div>
+{% endblock %}
+
 {% block company %}
 
 <div class="section is-medium has-text-centered">
     <h1 class="title">
-        <a href="{{ url_for('main_company.my_accounts') }}">Accounts</a> > Account r
+        Account r
```
Comment (seppeljordan) [inline arbeitszeit_flask/templates/company/account_r.html:12]: translation

File: arbeitszeit_flask/templates/company/account_a.html [hunk 9]
```diff
@@ -1,10 +1,15 @@
...
+<a class="navbar-item" href="{{ url_for('main_company.my_accounts') }}">{{ gettext("Accounts") }}</a>
+<div class="navbar-item">{{ gettext("Account a") }}</div>
+{% endblock %}
+
 {% block company %}
 
 <div class="section is-medium has-text-centered">
     <h1 class="title">
-        <a href="{{ url_for('main_company.my_accounts') }}">Accounts</a> > Account a
+        Account a
```
Comment (seppeljordan) [inline arbeitszeit_flask/templates/company/account_a.html:12]: translation

**Review by seppeljordan — APPROVED** [review]
I would like to see no new lines of code added with untranslatable strings. If you don't feel like marking those strings in the current PR, that's fine, but I think that we should not make it a habit to work on user facing strings and not translating them.

I marked to all strings that I found that were touched and not translated for documentation purposes so we can translate them later.

Certs transferred

**Comment by sloschert** [comment]
> I would like to see no new lines of code added with untranslatable strings. If you don't feel like marking those strings in the current PR, that's fine, but I think that we should not make it a habit to work on user facing strings and not translating them.
> 
> I marked to all strings that I found that were touched and not translated for documentation purposes so we can translate them later.
> 
> Certs transferred

OK. I made most of the strings you marked translatable in my last PR. The rest (2 files) are marked as well in issue #312.

**Pull request merged**